src/mlpack/core/data/save.hpp

```diff
@@ -59,6 +59,12 @@ bool Save(const std::string& filename,
           const bool fatal = false,
           bool transpose = true);
 
+template<typename eT>
+bool Save(const std::string& filename,
+          const arma::SpMat<eT>& matrix,
+          const bool fatal = false,
+          bool transpose = true);
```

Comment:
Can you add comprehensive documentation to this?  This is a user-facing function, so it's really important that it's documented very clearly. :+1:

Reply:
Looks good, thank you. :)